dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/singleevent/SingleEventOperationParamsMapper.java

```diff
@@ -98,6 +98,16 @@ public SingleEventQueryParams map(
     mapDataElementFilters(queryParams, operationParams.getDataElementFilters());
     mapOrderParam(queryParams, operationParams.getOrder());
 
+    if (operationParams.isSynchronizationQuery()) {
```

Comment:
With all the changes in `SingleEventOperationParams`, can we remove all these `if`s and just pass to the query params the value of the parameters?

Reply:
Or we can just remove this whole block of code?